src/aind_data_transfer_service/models/core.py

```diff
@@ -129,7 +129,6 @@ def __init__(self, /, **data: Any) -> None:
     model_config = ConfigDict(use_enum_values=True, extra="ignore")
 
     job_type: str = Field(
```

Comment:
To be consistent with the other fields, we should add ellipsis to indicate that it is required:
```python
    job_type: str = Field(
        ...,
        description=(
            "Job type for the upload job. Tasks will be run based on the "
            "job_type unless otherwise specified in task_overrides."
        ),
        title="Job Type",
    )
```